week1/week1.java

```diff
@@ -0,0 +1,72 @@
+public class week1 {
+
+	//////////Global Variables//////////
+
+	//////////Primitive Data Types//////////
+	public String myName = "George Burdell";
+	public char middleInitial = 'P';
+	//This is a comment!
+	/*
+	This is a block comment...
+	By the way there are more number types like
+	byte, short, long, and float
+	but you'll run into int and double the most
+	*/
+	public int yearsInRoboJackets = 3;
+	public double myBirthday = 8.11;
+	public Boolean isRoboJacketsCool = true;
```

Comment:
If you are trying to say that `Boolean` is passed by reference, you might want to add a comment here. Otherwise, change it to the primitive.